Sources/PopupBridge/POPPopupBridge.swift

```diff
@@ -160,22 +145,47 @@ public class POPPopupBridge: NSObject, WKScriptMessageHandler, SFSafariViewContr
                 return
             }
             
-            if let urlString = script.url,
-               let url = URL(string: urlString) {
-                dismissSafariViewController()
-                
-                delegate.popupBridge?(self, willOpenURL: url)
-                
-                let viewController = SFSafariViewController(url: url)
-                safariViewController = viewController
-                safariViewController?.delegate = self
-                
-                self.delegate.popupBridge(self, requestsPresentationOfViewController: viewController)
-                return
+            if let urlString = script.url, let url = URL(string: urlString) {
+                webAuthenticationSession.start(url: url, context: self) { url, error in
+                    if let error = error as? NSError {
+                        if error.code == ASWebAuthenticationSessionError.canceledLogin.rawValue {
```

Comment:
Can we move the cancel logic into it's own callback param like we did on BT?

Reply:
👍 https://github.com/braintree/popup-bridge-ios/pull/57/commits/abef023b316916f5da4bdbd38626214e9abb3af6